test/Date/xplatInterval.js

```diff
@@ -8,13 +8,20 @@ if (WScript.Platform && WScript.Platform.OS != "win32") {
     function rand() {
         return parseInt(Math.random() * 1e2) + 50;
     }
+
     for (var j = 0; j < 1e2; j++) {
         var pre_time = Date.now(), now;
         for(var i = 0; i < 1e6; i++) {
             now = Date.now();
             var diff = now - pre_time
 
-            if (diff < 0) throw new Error ("Timer interval has failed. diff < 0");
+            // INTERVAL_FOR_TICK_BACKUP = 5
+            // So, anything beyond 5ms is not subject to our testing here.
```

Comment:
This scenario makes sense but isn't system timer granularity is somewhere around 10ms? I'm not saying we can't see that test is failing but we most likely don't and I prefer to see that test is failing on different environments instead of discarding the comparison whenever the system timer backwards.